Engine/gfx/ali3dogl.h

```diff
@@ -255,6 +255,7 @@ class OGLGraphicsDriver : public VideoMemoryGraphicsDriver
     bool _smoothScaling;
     bool _legacyPixelShader;
 
+    bool _shaders_created {};
```

Comment:
What does this `{}` do in declaration? I noticed nick sonneveld was adding these too, but not for all the members.
Perhaps I'm missing something about the new C++ standard.

Reply:
It's the default initialization, in this case means false. I can rewrite so it's written false instead.

Reply:
Tbh I am not well aware of how this works, in which circumstances will this value be applied?
The class already has a defined constructor, is this default initialization happens before the constructor?

Reply:
PS. that's not a problem for this PR, I was rather curious.

Reply:
I found a bit of writing on this here: https://en.cppreference.com/w/cpp/language/value_initialization

If there's a value in the list initializer then it doesn't do narrowing conversion.